src/settings/LocationLibraries.test.js

```diff
@@ -133,4 +134,20 @@ describe('LocationLibraries', () => {
 
     userEvent.click(screen.getByTestId('button-new'));
   });
+
+  it('should render LocationLibraries with filled form', async () => {
+    renderLocationLibaries(resourcesMock);
+
+    const checkboxInstitution = screen.getByRole('combobox', { name: 'ui-tenant-settings.settings.location.institutions.institution' });
+
+    await userEvent.selectOptions(checkboxInstitution, 'Københavns Universitet (KU)');
+
+    expect(screen.getByRole('option', { name: 'Københavns Universitet (KU)' }).selected).toBe(true);
+
+    const checkBoxCampuses = screen.getByRole('combobox', { name: 'ui-tenant-settings.settings.location.campuses.campus' });
+
+    await userEvent.selectOptions(checkBoxCampuses, 'City Campus (CC)');
+
+    userEvent.click(screen.getByTestId('1'));
```

Comment:
A test should end with an assertion, not an action. Here, you're basically calling a function but not checking its result.